### ISSUE TITLE
Please add more verbose download details for various survey data sets (or any other DLC)

### ISSUE THREAD

**Issue by ckuethe** [Stellarium/stellarium#2587]
**Is your feature request related to a problem? Please describe.**
I'm trying to load a few more types of survey images and the download appears to hang.

**Describe the solution you'd like**
Log messages describing what is currently being downloaded, possibly in a way that I could reproduce with curl/wget/axel/etc.

**Describe alternatives you've considered**
Tried running with multiple verbose flags in case this information is logged, but only at verbose log level: `~/AppImage/Stellarium-0.22.2-x86_64.AppImage --verbose --verbose --verbose`

**Additional context**
The only relevant log entry is
`Downloaded 47 files (4560 kbytes) in a session of 843.601 sec (average of 5.4054 kB/s + 1080 files from cache (5117 kB)).`

No indication of what index images are being downloaded, from where, or how many are to be downloaded.

**Comment by gzotti** [comment]
This message is a summary of all data updates, e.g. satellites, meteor shower data or novae. The HiPS surveys don't count here. Those are described in chapter 10 of the User Guide.

**Comment by ckuethe** [comment]
Right. My point is that the progress bar seemed to stall about half way through various online survey retrievals (hubble, JWST, iras) and there was nothing in the logs indicating what exactly was going on:
* what url was being fetched?
* was the download suffering from some network problems? How long was it before the download was retried?
* what is the total download size?
* other details that might be useful for troubleshooting...
